devtools/common/src/main/java/io/quarkus/cli/commands/file/GradleBuildFile.java

```diff
@@ -81,6 +82,14 @@ private void completeBuildContent(String groupId, String version) {
         if (!buildContent.contains("apply plugin: 'io.quarkus'") && !buildContent.contains("id 'io.quarkus'")) {
             res.append(System.lineSeparator()).append("apply plugin: 'io.quarkus'").append(System.lineSeparator());
         }
+        if (!buildContent.contains("apply plugin: 'java'")) {
+            res.append(System.lineSeparator());
+            res.append("apply plugin: 'java'").append(System.lineSeparator());
```

Comment:
Are you certain we need `apply plugin: 'java'`? Does it work without it, I mean by just adding the `compileJava` Gradle task with the configuration you have supplied?

Reply:
Actually I am almost certain we shouldn't do `apply plugin: 'java'` since the `java` plugin has is being applied using:

```
plugins {
    id 'java'
}
```

Reply:
yes, I verified also. I'm going to do the change on the code

Reply:
cool